code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm

```diff
@@ -151,46 +151,24 @@
 	return TRUE
 
 
-//Sigil of Transmission: Creates a sigil of transmission that can store power for clockwork structures.
+//Sigil of Transmission: Creates a sigil of transmission that can drain and store power for clockwork structures.
 /datum/clockwork_scripture/create_object/sigil_of_transmission
-	descname = "Structure Battery"
+	descname = "Structure Power Generator & Battery"
 	name = "Sigil of Transmission"
-	desc = "Places a sigil that stores energy to power clockwork structures."
+	desc = "Places a sigil that can drain and will store energy to power clockwork structures."
 	invocations = list("Divinity...", "...power our creations!")
 	channel_time = 70
 	consumed_components = list(VANGUARD_COGWHEEL = 2, GEIS_CAPACITOR = 2, HIEROPHANT_ANSIBLE = 4)
```

Comment:
Daily reminder that if this gets merged like so it'll lower the cost for draining energy form 2 Vanguard Cogwheels, 5 Eyes, 2 Geis Capacitors, 2 Alloys, 6 Heirophants total to only 2 Cogwheels, 2 geis capacitors and 4 hierophant ansibles.
Update when